lib/local_daily_digest.js

```diff
@@ -0,0 +1,167 @@
+var async = require('async'),
+	_ = require('lodash'),
+	log = require('../log'),
+	Profile = require('../models').Profile,
+	Contact = require('../models').Contact,
+	log = require('../log'),
+	mail = require('../mail');
+
+function get(req, res, next) {
+	var result = {},
+	 	allLocalProfilesWDigest = [],
+	 	allLocalProfiles =[],
+	 	requiredUsers=[],
+	 	checkedOutUsers = [],
+	 	finalCheckedOutList=[],
+		tempManagers =[],
+		tempUsers = [],
+		managerContactsProfile = [],
+		finalOutput = [],
+		tempProf=[],
+		requiredLocationsId =[],
+		requiredLocationsName =[],
+		temp='',
+		tempProfile ='',
+		checkedInUsers = [];
+	
+	async.series([
+		//THe local daily digest will be just the digest for that country !!
+		function (cb) {
+			Profile.find({'dailyDigest': {$not: {$size: 0}}}, function(err, profile){
+			if(err){
+		  		res.send(500, new Error(err));
+		  		return cb(true);
+		  	}
+
+		  	if(!profile){
+		  		res.send(400, new Error('No Profiles were found'));
+		  	}
+
+		  	profile.forEach(function(profile){
+			  		allLocalProfilesWDigest.push(profile);
+			  	});
+	  			return cb();
+			});
+		},
+		function(cb){
+			allLocalProfilesWDigest.forEach(function(profile){
+				// console.log(profile);
+				profile.dailyDigest.forEach(function(location){
+					requiredLocationsId.push(location);
+				});
+				// requiredLocationsName.push(profile.location);
+			});
+			console.log("the countries this profile is either a manger or editor of are: ", requiredLocationsId);
```

Comment:
Yeah sorry, I added that check in the hid-app today. The countries are selected from the daily digest array in hid-profiles. 
In the app, the user is only able to select from a list of countries he/she is a manager or editor off. That check is done in the app. 

The other thing I can do, is that a user can only select countries in the app, if the user is verified. If they are not verified, they wont have the option to select countries for the digest, no matter if they are an editor/manager